### ISSUE TITLE
Cheat Prevention. Delete movie feature

### ISSUE THREAD

**Issue by Enddox** [RetroAchievements/RAEmus#30]
### Delete The Movie Feature

Delete the File -> Movie From the RAnes because you can easily cheat recording a movie and playing it while you are in hardcore mode, I don't know if the movie feature its on RASnes9x, RAGens or another emulator but having this feature makes the life easy for the cheaters

:(

**Comment by kdecks** [comment]
The movie features is restricted to Softcore, which is more permissible in general, allowing saves states, rewind, slow down among other things.

**Comment by Enddox** [comment]
> The movie features is restricted to Softcore, which is more permissible in general, allowing saves states, rewind, slow down among other things.

You can play a movie on hardcore going to File -> Movie -> Recent -> Select Your movie

**Comment by kdecks** [comment]
Absolutely. Now try doing it with hardcore enabled.

**Comment by Jamiras** [comment]
No, he's right. Play is disabled, but the recent list is not. I'll fix it.

**Comment by Enddox** [comment]
> Absolutely. Now try doing it with hardcore enabled. One reason this could be permissible in softcore is it can be a useful developer tool.
> ![image](https://user-images.githubusercontent.com/32706333/51080602-36b43600-169c-11e9-9b21-778d69ebb305.png)

http://prntscr.com/m6gl9q

**Comment by kdecks** [comment]
Good catch
